packages/combobox/src/Combobox.types.ts

```diff
@@ -317,10 +330,12 @@ export interface OptionObject {
   displayName: string;
   isDisabled: boolean;
   hasGlyph?: boolean;
+  description?: string;
+  onClick?: React.EventHandler<OptionClickEvent>;
 }
 
 export interface InternalComboboxOptionProps
-  extends ComponentPropsWithoutRef<'li'> {
+  extends Omit<ComponentPropsWithoutRef<'li'>, 'onClick'> {
```

Comment:
should we be reading the base type from the external definition? or vice versa (rather than redefining)